hoot-core/src/main/cpp/hoot/core/elements/WayData.h

```diff
@@ -56,6 +56,7 @@ class WayData : public ElementData
   virtual ~WayData();
 
   void addNode(long id) { _nodes.push_back(id); }
+  void insertNode(long index, long id ) { _nodes.insert(_nodes.begin()+index, id); }
```

Comment:
This is just a small nit-pick but we always put spaces between operators and operands. `_nodes.begin() + index`.

Reply:
Fixed